packages/@okta/vuepress-site/docs/guides/single-logout/main/index.md

```diff
@@ -43,18 +43,18 @@ Ideally, when the user wants to sign out, they should sign out of every app to k
 
 </div>
 
-**Event 1**
+**Event 1:**
 
-* The user signs out of App 1 using Browser 1.
-* App 1 initiates the logout (SP-initiated) by sending a front-channel inbound logout request to Okta using Browser 1. For example:
+* The user signs out of App 1 using browser 1.
```

Comment:
these coincide with the image: Browser 1, Browser 2 and so on - Session 1, Session 2, and so on. Can you change them back, please?